### PR TITLE
Adds coveralls for ci, adds coverage folder to gitignore

### PR DESCRIPTION
**Pull request by GergKllai1**
# Features:
- Adds coveralls for CI

## PT Links:
https://www.pivotaltracker.com/story/show/163088235

### CODE REVIEW

File: lib/tasks/ci.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    require 'cucumber/rake/task'
+    require 'coveralls/rake/task'
+  
+    Coveralls::RakeTask.new
+  
+    namespace :ci do
+      desc 'Run all tests and generate a merged coverage report'
+      task tests: [:spec, :cucumber, 'coveralls:push']
+    end
+  end
```
Comment (oliverochman) [inline lib/tasks/ci.rake]: fix indentation here

File: features/support/env.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 require 'cucumber/rails'
+require 'coveralls'
```
Comment (oliverochman) [inline features/support/env.rb:1]: put line 2 & 3 at the top of the file

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: features/support/env.rb (hunk 2 above)
Comment (GergKllai1) [reply features/support/env.rb:1]: done

File: lib/tasks/ci.rake (hunk 1 above)
Comment (GergKllai1) [reply lib/tasks/ci.rake]: done

**Review by oliverochman — APPROVED** [review]
LGTM!

**Pull request merged**